### DISCUSSION TITLE
Could it support OAuth authentication

### DISCUSSION THREAD

**Discussion by ghost** [mongo-express/mongo-express#1003] [Ideas]
In fact I love this GUI but it seems don't support OAuth authentication.
If I am wrong, could it be connected to any OAuth authentication like OKTA, Google OAuth, OpenID.
As many user might has its own authentication system but would like to use your mongo express project intergated to their side.

**Comment by rtritto** [comment]
Done in #1666
Released in [v1.1.0-rc-3](https://github.com/mongo-express/mongo-express/releases/tag/v1.1.0-rc-3)

**Comment by rtritto** [comment]
FYI @SleepyDeb
